apps/davi/src/stores/modules/1_5/fetchers/subgraph/useVotingPowerOf/useVotingPowerOf.ts

```diff
@@ -25,16 +28,31 @@ export const useVotingPowerOf: IUseVotingPowerOf = ({
     }
   );
 
-  const userVotingPower = useMemo(() => {
-    if (!data?.dao) return undefined;
-    const member = data?.dao?.reputationToken?.members?.[0];
+  const votingPowerOfResponse = useMemo(() => {
+    if (!data || !data.dao || !data.dao.reputationToken.members)
+      return undefined;
+    const member = data.dao.reputationToken.members?.[0];
 
     return member ? BigNumber.from(member.reputationTokenAmount) : undefined;
-  }, [data?.dao]);
+  }, [data]);
 
-  return {
-    data: userVotingPower,
-    isLoading: loading,
-    isError: !!error,
-  };
+  const currentSnapshotId = data?.dao?.reputationToken?.currentSnapshotId;
+
+  if (!snapshotId) snapshotId = currentSnapshotId;
+
+  const votingPowerAtSnapshotResponse = useVotingPowerOfAt(
+    daoId,
+    userAddress,
+    fallbackSnapshotId ? snapshotId ?? currentSnapshotId : snapshotId

```

Comment:
I don't understand this line.

`snapshotID` will either be assigned to the hook param or `currentSnapshotId`, so there's no chance for it to be null.

My understanding is that this line will always return `snapshotId`